changelog.md

```diff
@@ -4,11 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.4] - 2020-03-06
+## [unreleased] - 2020-05-05
```

Comment:
I'm a bit confused about the versioning for this buildpack. The GitHub releases/tags use a `1.x` versioning scheme, vs the buildpack releases versioning which uses `vN` (not that there is a choice).

Also the latest GitHub release/tag is on 2019-08-09, but there was a buildpack release in Jan:

```
$ h buildpacks:versions heroku-community/nginx
Version  Released At               Status
───────  ────────────────────────  ─────────
5        2020-01-17T11:59:57.362Z  published
4        2019-08-09T07:39:23.045Z  published
3        2019-05-29T11:31:56.786Z  published
2        2019-05-14T20:41:25.605Z  published
1        2018-09-19T09:12:41.784Z  published
```

Reply:
i'll merge the PR's, update the the changelog file and release a new version to get us back on track